pkgs/development/compilers/cudatoolkit/flags.nix

```diff
@@ -8,7 +8,8 @@
 #   archName: String, # e.g., "Hopper"
 #   computeCapability: String, # e.g., "9.0"
 #   minCudaVersion: String, # e.g., "11.8"
-#   maxCudaVersion: String, # e.g., "12.0"
+#   dontDefaultAfter: null | String, # e.g., null
```

Comment:
`minCudaVersion` and `maxCudaVersion` seem pretty self explanatory. But what does `dontDefaultAfter` mean exactly? Could we add a comment clarifying what this does and why it's necessary?

Also are `minCudaVersion` and `maxCudaVersion` inclusive or exclusive constraints? I'm guessing inclusive but better to be explicit.

Reply:
You're correct, they're inclusive. I'll add some comments in `gpus.nix` which better describe them!